tfhe/c_api_tests/test_high_level_zk.c

```diff
@@ -8,7 +8,7 @@ int main(void) {
 
   // Note that simply changing parameters like this does not yield secure parameters
   // Its only done for the example / tests
-  ShortintPBSParameters params = SHORTINT_PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64;
+  ShortintPBSParameters params = SHORTINT_PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM;
```

Comment:
no more 2M64 ?

Reply:
I use an alias since 2M64 is the default p-fail value.